src/libs/database/impl/Release.cpp

```diff
@@ -167,10 +167,10 @@ namespace lms::db
             case ReleaseSortMethod::None:
                 break;
             case ReleaseSortMethod::Name:
-                query.orderBy("r.name COLLATE NOCASE");
+                query.orderBy("COALESCE(r.sort_name, r.name) COLLATE NOCASE");
```

Comment:
I have a lot of albums without sort_name and it does not seem to work well.
Unfortunately, `sort_name `is always stored as non null, so "" is always taken into account for sort_name.
So first option would be to store sort_name using std::optional<std::string> (and handle migration)
Second option would be to change the query to something like `CASE WHEN sort_name <> '' THEN sort_name COLLATE NOCASE ELSE name COLLATE NOCASE END`

Reply:
Aaargh, I didn't notice the "" thing because _all_ my albums have the `ALBUMSORT` tag 😅 

I'll try the way of the optional ™️ in the next days.

Reply:
No problem, thanks again for the PR :)
The std::optional way is indeed the best choice. Don't hesitate to tell me if you need help! x